src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -94,6 +94,46 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task HttpHeader_LastModifiedDate_Success()
+        {
+            DateTime expected = TimeZoneInfo.ConvertTimeFromUtc(new DateTime(2018, 4, 10, 3, 4, 5, DateTimeKind.Utc), TimeZoneInfo.Local);
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(url);
+                request.Method = HttpMethod.Get.Method;
```

Comment:
I think the default of HttpWebRequest.Method is "GET", you probably don't need this line.